bin/v-add-web-domain

```diff
@@ -94,6 +94,7 @@ chown root:$user /var/log/$WEB_SYSTEM/domains/$domain.* $conf
 chmod 640 /var/log/$WEB_SYSTEM/domains/$domain.*
 chmod 751 $HOMEDIR/$user/web/$domain $HOMEDIR/$user/web/$domain/*
 chmod 551 $HOMEDIR/$user/web/$domain/stats $HOMEDIR/$user/web/$domain/logs
+chmod 755 $HOMEDIR/$user/web/public_*html
```

Comment:
Shouldn't this not be  $HOMEDIR/$user/web/$domain/public_*html ?

$HOMEDIR/$user/web/public_*html does not exists...